### DISCUSSION TITLE
Warning about ACR to LoA Mapping, although no entries defined

### DISCUSSION THREAD

**Discussion by 0xpr03** [keycloak/keycloak#13946] [Q&A]
My keycloak 19.0.1 logs (from an imported realm) complain about
```
WARN  [org.keycloak.protocol.oidc.utils.AcrUtils] (executor-thread-183) Invalid realm configuration (ACR-LOA map)
```

regularly.

![image](https://user-images.githubusercontent.com/1280209/186268998-224f04a8-2f02-46b9-9b12-2f8fc8d3f2ac.png)

As you can see I don't define any mapping..

How can I get rid of this message ?

**Comment by connium** [comment]
I was able to fix this problem by removing the `"acr.loa.map" : "[]"` entry from the realm.json that I use for the import.

**Reply by 0xpr03** [reply]
That's nice to hear. Sadly I don't want to compeltely erase and re-install my realm, also due to the documented failures in correctly importing keys.
